userspace/files/adbd.service

```diff
@@ -3,4 +3,4 @@ Description=adbd Service
 
 [Service]
 User=root
-ExecStart=/usr/comma/adbd
+ExecStart=/bin/sh -c 'export TERM=xterm-256color; su - comma -c "cd /data/openpilot && exec /usr/comma/adbd"'
```

Comment:
This does not work. Also, it should not assume the existence of `/data/openpilot` (for example of first boot)